bin/submit_order.rs

```diff
@@ -103,7 +103,10 @@ async fn send_order(
     let send_order = SendOrder::new(signer.clone(), config.constants.clone())?;
 
     // sign the order, return it back for comparison
-    let signed = order.sign(signer).await?;
+    let signed = order
+        .with_chain(config.constants.system())
+        .sign(signer)
+        .await?;
```

Comment:
i'm a bit confused about this. if there's no chain id set, shouldn't signing the order outright fail? how was this working before?